### PR TITLE
[Merged by Bors] - documented min. operator requirements

### LINKED ISSUE #559: Document operator system requirements
We want to document user-_guidelines_ for the operator in terms of e.g. min/max CPU, memory and disk space. These are **non-binding** (as an accurate assessment is dependent on a specific use-case) but may be referred to from GTCs and other contract documents. 

This should be documented as described in https://github.com/stackabletech/documentation/issues/289

## Acceptance

- [x] run the following
  - [x] zookeeper operator managing single ZK instance (serving HDFS, HBase and Kafka clusters)
  - [x] zookeeper operator managing multiple ZK instances in different namespaces (each serving HDFS, HBase and Kafka clusters)
  - [x] vary cluster sizes 
- [x] recommendations for CPU, memory and disk space per zookeeper instance managed by the operator, for
  - [x] Kind cluster 
  - [x] Managed k8s cluster (at least one)
- [x] The results are added to a table in the documentation as described in https://github.com/stackabletech/documentation/issues/289

### PR DESCRIPTION
**Pull request by adwk67**
# Description

Documented: approximate min. operator system requirements recommendation.
Closes #559 

## Review Checklist

- [ ] Code contains useful comments
- [ ] CRD change approved (or not applicable)
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
- [ ] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/getting_started/pages/index.adoc [hunk 1]
```diff
@@ -10,6 +10,11 @@ You will need:
 * kubectl
 * optional: Helm
 
+Resource sizing depends on cluster type(s), usage and scope, but as a starting point we recommend a minimum of the following for this operator:
```
Comment (maltesander) [inline docs/modules/getting_started/pages/index.adoc:13]:
```suggestion
Resource sizing depends on cluster type(s), usage and scope, but as a starting point we recommend a minimum of the following resources for this operator:
```

**Comment by adwk67** [comment]
> One question: We will use theses values for all operators or do individual tests? If individual, i would document the used values here [stackabletech/documentation#289](https://github.com/stackabletech/documentation/issues/289) as well.

I think the plan is to take this benchmark and use it for all operators, but to document it (using the same text) for all operators individually under "prerequisites" in the installation section. That way we can always adjust per operator in the future if we need finer granularity.

**Review by maltesander — APPROVED** [review]
LGTM!

**Comment by adwk67** [comment]
bors merge

**Pull request closed without merging**